package.json

```diff
@@ -14,7 +14,12 @@
     "watch": "npm run build -- --watch",
     "start": "webpack-dev-server --config examples/using-module/webpack.config.babel.js --context examples/using-module --open",
     "analyze": "webpack --profile --json --progress > stats.json",
-    "lint": "eslint . --cache && stylelint **/*.scss --cache"
+    "lint-js": "eslint . --cache",
+    "lint-sass": "stylelint **/*.scss --cache",
```

Comment:
I'm pretty sure the only sass is in one of your other PRs, not in the app yet

Reply:
Good point!